src/rulesets/LinearAlgebra/utils.jl

```diff
@@ -27,3 +27,5 @@ function _eyesubx!(X::AbstractMatrix)
 end
 
 _extract_imag(x) = complex(0, imag(x))
+
+_unionallof(::Type{T}) where {T} = T.name.wrapper
```

Comment:
This name is not obvious to me why we say that.
would
`_unionall_wrapper` be clearear?
Can we also add a docstring?

Reply:
> would
> `_unionall_wrapper` be clearear?

Yeah naming this is tricky. It's basically `Base.typename` but instead of returning a `TypeName` it returns a `UnionAll`. If `_unionall_wrapper` is clearer, I can do that.